app/serializers/activitypub/note_serializer.rb

```diff
@@ -39,11 +41,11 @@ def summary
   end
 
   def content
-    Formatter.instance.format(object)
+    status_content_format(object)
```

Comment:
Maybe memoize for perfs?

Reply:
You cannot memoize in a serializer because the same instance is reused for multiple objects.

Reply:
Oh, right. It would be good to not format it twice, but it wasn't prior to this PR and it is probably not worth it anyway.